listenbrainz/listenstore/timescale_utils.py

```diff
@@ -85,42 +277,16 @@ def add_missing_to_listen_users_metadata():
 
 
 def recalculate_all_user_data():
```

Comment:
Missing docs for this function.

Reply:
This function is intended to run only once, right?

Reply:
Yes. Only need to run this when we we want to recalculate everything from scratch which should be only when we deploy this unless something goes wrong.